SPL-Challenges.tex

```diff
@@ -121,4 +121,6 @@ \subsection{Code Publication}
 
 \include{challenges/shared_autonomy_challenge}
 
+\include{challenges/leaderboard_challenges}
```

Comment:
I think Leaderboard should be it's own documents. 
Technical are likely different every years, but the idea of leaderboard is to have tests with minimum update year to year.